pep-0590.rst

```diff
@@ -121,13 +133,29 @@ For getting the actual number of arguments from the parameter ``n``,
 the macro ``PyVectorcall_NARGS(n)`` must be used.
 This allows for future changes or extensions.
 
+Finalizing the API
```

Comment:
This section should logically come *after* the section where you introduce `_PyObject_Vectorcall`.